config/checkstyle/checkstyle.xml

```diff
@@ -118,7 +118,11 @@
 
         <!-- Checks for Size Violations. -->
         <!-- See http://checkstyle.sf.net/config_sizes.html -->
-        <module name="MethodLength" />
+
+        <module name="MethodLength">
+            <property name="max" value="400" />       
+        </module>
+        
```

Comment:
Such empty line with trailing space should be eliminated in code - please fix here and elsewhere.